src/lib.rs

```diff
@@ -2133,13 +2255,21 @@ pub struct PublishDiagnosticsParams {
 
     /// An array of diagnostic information items.
     pub diagnostics: Vec<Diagnostic>,
+
+    /// Optional the version number of the document the diagnostics are published for.
+    pub version: Option<i64>,
```

Comment:
I think there's a `Number` field or typedef somewhere in this crate that you can use.

Reply:
And of course #[serde(skip_serializing_if = "Option::is_none")]